corehq/apps/app_manager/static/app_manager/js/details/case_claim.js

```diff
@@ -203,9 +203,14 @@ hqDefine("app_manager/js/details/case_claim", function () {
         };
 
         subscribeToSave(self, searchConfigKeys, saveButton);
+        // media image/audio buttons
         $(".case-search-multimedia-input button").on("click", function () {
             saveButton.fire('change');
         });
+        // checkbox to select media for all languages
+        $(".case-search-multimedia-input input[type='checkbox']").on('click', function () {
```

Comment:
I think this should be `change` because `click` won't notice if a user toggles the checkbox with their space bar.

Reply:
https://github.com/dimagi/commcare-hq/pull/29765/commits/05a49f1d95e86b554b7679c6e29fa45428e3e240

I would not have thought of anyone using spacebar + tabs to get to the checkbox. took me a few tabs to get to the checkbox. :)